core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/catalog/dataformats/DataFormatModel.java

```diff
@@ -1,103 +0,0 @@
-/*******************************************************************************
```

Comment:
why did u delete that class?

Reply:
It's not used anywhere. Not in tests. Not in code. Nowhere.

Reply:
wow, that seems to be a zombie from the xml model days....wondering how and why it survived.

Reply:
Not sure, but definitely not needed. :)